### DISCUSSION TITLE
monitoring page

### DISCUSSION THREAD

**Discussion by dbekaert** [nasa/opera-sds#59] [Ideas]
Briefly wanted to start a discussion channel for the features for the monitoring page. Tagging a few people @piyushrpt @alhandwerger. Also, this would be some sort of resource to be placed in close proximity of the table we are building for the PGE's/ Dockers/ Ancillaries and where to access them. The internal page can show more information on also pricing etc.

Simplistically i would like to be able to convey product status:
- a summary table:
--  traffic light status for the products based of historical expectation e.g. are product being products within 1 sigma of mean for last 15-30 days or so. The summary i would also let it contain some metric information. 
-- At min the average TOTAL latency (read sensor acquisition to product available at DAAC).
-- URL to access the data in earthdatasearch
-- URL to access product landing page at the DAAC

- Additionally there could be a set of charts with higher level information to dive into which could be the time-series block diagram which could show the average and 1-sigma etc show to visually represent  the traffic light. These are pretty much the ones we already have on the portal with some modifications.

One topic is for the charts as well as the numbers that are being used in the charts to be for fixed periods. e.g. if you go day by day then dont show 1 hour of a new day compared to full days. 

<img width="1034" alt="Screenshot 2024-10-11 at 4 05 15 PM" src="https://github.com/user-attachments/assets/a01883cc-c631-42e6-a797-210606116360">

**Comment by LucaCinquini** [comment]
Jim already made some changes to the script, see latest output. Next he is going to work on filtering the inputs, specifically:

HLSL30: filter out Landsat 9 in the inputs.
HLSS30: Add a filter that includes only S2A, clarify in legend.

After that, we can start working on computing the latency.
